security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-3-0"]
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Compliance Operator 1.3.0 now supports creation of the DISA-STIG profile within the supported profiles module for {product-title} 4.14. See https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles
+ for more information.
+
+* Compliance Operator 1.3.0 now provide support to IBM Power and IBM Z for NIST moderate profiles. 
++
+[IMPORTANT]
```

Comment:
This is probably okay to add, as users may still be upgrading from versions prior to 1.2.0. We should run this past engineering and QE to be sure. We may need to rewrite, something like "If you are upgrading from a version prior to CO 1.2.0, ..." etc.

Reply:
I'll check that with eng and QE,

Reply:
I've deleted the IMPORTANT block per SME advisement. Thank you!